### ISSUE TITLE
Glances crashes after a variable and unpredictable period of time (KeyError: 'memory_info')

### ISSUE THREAD

**Issue by piotrp88** [nicolargo/glances#1303]
#### Description

Glances crashes after randomly and unpredictably. This happens two or three time in a day usually (I leave glances running all day on a terminal while working). Sometimes it crashes after ten minutes, sometimes after some hours. The error is always the same (see traceback at the end of this message). I run glances as root.

#### Versions

* Glances & psutil (glances -V): `Glances v2.11.1 with psutil v5.4.3`
* Operating System (lsb_release -a):

```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial
```

#### Logs

```
Traceback (most recent call last):
  File "/usr/local/bin/glances", line 9, in <module>
    load_entry_point('Glances==2.11.1', 'console_scripts', 'glances')()
  File "/usr/local/lib/python3.5/dist-packages/glances/__init__.py", line 134, in main
    start(config=config, args=args)
  File "/usr/local/lib/python3.5/dist-packages/glances/__init__.py", line 103, in start
    mode.serve_forever()
  File "/usr/local/lib/python3.5/dist-packages/glances/standalone.py", line 117, in serve_forever
    self.schedule.run()
  File "/usr/lib/python3.5/sched.py", line 147, in run
    action(*argument, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/glances/standalone.py", line 104, in __serve_forever
    self.screen.update(self.stats)
  File "/usr/local/lib/python3.5/dist-packages/glances/outputs/glances_curses.py", line 978, in update
    self.flush(stats, cs_status=cs_status)
  File "/usr/local/lib/python3.5/dist-packages/glances/outputs/glances_curses.py", line 956, in flush
    self.display(stats, cs_status=cs_status)
  File "/usr/local/lib/python3.5/dist-packages/glances/outputs/glances_curses.py", line 528, in display
    __stat_display = self.__get_stat_display(stats, plugin_max_width)
  File "/usr/local/lib/python3.5/dist-packages/glances/outputs/glances_curses.py", line 492, in __get_stat_display
    ret[p] = stats.get_plugin(p).get_stats_display(args=self.args)
  File "/usr/local/lib/python3.5/dist-packages/glances/plugins/glances_plugin.py", line 762, in get_stats_display
    'msgdict': self.msg_curse(args),
  File "/usr/local/lib/python3.5/dist-packages/glances/plugins/glances_processlist.py", line 461, in msg_curse
    self.__msg_curse_sum(ret, args=args)
  File "/usr/local/lib/python3.5/dist-packages/glances/plugins/glances_processlist.py", line 527, in __msg_curse_sum
    msg = '{:>6}'.format(self.auto_unit(self.__sum_stats('memory_info', indice=1, mmm=mmm), low_precision=False))
  File "/usr/local/lib/python3.5/dist-packages/glances/plugins/glances_processlist.py", line 616, in __sum_stats
    ret += p[key][indice]
KeyError: 'memory_info'
```

**Comment by nicolargo** [comment]
Hi @piotrp88 

It should be solved in the DEVELOP branch.

Can you test it on your system ?

Nicolas

**Comment by piotrp88** [comment]
I started a glances session using the DEVELOP branch. I forgot to mention that I always run glances with `glances -t 1`.

I will see what happens in the next hours.

PS in the DEVELOP version the RAID section doesn't show up.

**Comment by piotrp88** [comment]
I used glances all day yesterday and it didn't crash. I observed a higher CPU usage compared to the stable version (near 100% on a single core), even without the `-d` flag. The left column is too big in my opinion, while on the stable version it has the perfect width. See the attached images.

![devel](https://user-images.githubusercontent.com/5948581/43942602-fe9d952e-9c78-11e8-91ff-50ae008dd433.png)

![stable](https://user-images.githubusercontent.com/5948581/43942605-01768382-9c79-11e8-9ea5-cd4d981206ba.png)
